<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDatabaseDialog</class>
 <widget class="QDialog" name="ConnectDatabaseDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>277</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>DatabaseType</string>
       </property>
       <property name="buddy">
        <cstring>databaseTypeBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="databaseTypeBox">
       <item>
        <property name="text">
         <string>QPSQL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QSQLITE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QMYSQL</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hostname</string>
       </property>
       <property name="buddy">
        <cstring>hostNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostNameEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DatabaseName:</string>
       </property>
       <property name="buddy">
        <cstring>databaseNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="databaseNameEdit">
       <property name="text">
        <string>imageset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>User name:</string>
       </property>
       <property name="buddy">
        <cstring>userNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userNameEdit">
       <property name="text">
        <string>lang</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>passwordEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="text">
        <string>lang</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Can&amp;cel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDatabaseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDatabaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
